src/js/components/ClipboardTrigger.js

```diff
@@ -44,39 +43,59 @@ class ClipboardTrigger extends React.Component {
     }
   }
 
+  getTriggerContent() {
+    const {children, className} = this.props;
+
+    if (children != null) {
+      return (
+        <span className={className} onMouseEnter={this.handleCopyIconMouseEnter}
+          ref="copyButton">
+          {children}
+        </span>
+      );
+    }
+
+    return (
+      <span onMouseEnter={this.handleCopyIconMouseEnter} ref="copyButton">
+        <Icon
```

Comment:
I did this, but this means we introduce an unnecessary render cycle for the `Icon` component in some cases. It's not a big deal in this component, but we should be cautious about doing this everywhere, IMO.